.github/workflows/R-CMD-check.yaml

```diff
@@ -0,0 +1,89 @@
+# For help debugging build failures open an issue on the RStudio community with the 'github-actions' tag.
+# https://community.rstudio.com/new-topic?category=Package%20development&tags=github-actions
+on:
+  push:
+    branches:
+      - main
+      - master
+  pull_request:
+    branches:
+      - main
+      - master
```

Comment:
@jmtaylor-fhcrc @wfulp - what are your thoughts on adding actions running on PRs to develop or pushes to develop?